tutorenrollmentreports/templates/enrollmentreports/build/enrollmentreports/Dockerfile

```diff
@@ -2,20 +2,12 @@ FROM python:3.8
 ENV PYTHONUNBUFFERED 1
 RUN python3 -m venv /enrollmentreports/venv/
 ENV PATH "/enrollmentreports/venv/bin:$PATH"
-RUN mkdir -p /etc/ansible/roles/enrollment_report && \
-    mkdir -p /etc/ansible/group_vars
+RUN mkdir -p /etc/ansible/roles/enrollmentreports /etc/ansible/group_vars
 RUN apt-get update && \
-    apt install ansible -y && \
-    apt-get install rsync -y && \
-    apt-get install default-mysql-client -y
+    apt install -y ansible rsync default-mysql-client
 
-RUN git clone https://github.com/hastexo/ansible-role-enrollmentreports.git /etc/ansible/roles/enrollment_report
+RUN git -C /etc/ansible/roles clone https://github.com/hastexo/ansible-role-enrollmentreports.git enrollmentreports
 RUN echo "postfix postfix/main_mailer_type string Internet site" > postfix_install.txt
-RUN echo "postfix postfix/mailname string mail.example.com" >> postfix_install.txt
 RUN debconf-set-selections postfix_install.txt
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -q -y postfix
 COPY enrollment-report.yml .
-COPY run_enrollment_reports.sh .
-COPY inventory/config.yml /etc/ansible/group_vars/all
-RUN  useradd mysql
```

Comment:
If I run the playbook, without adding `mysql` user explicitly I see the following Ansible file module error :
` chown failed: failed to look up user mysql`

Reply:
Yeah on second thought, it doesn't really make a ton of sense to base this image on `python:3.8`. See b94cd5156a6637dcfd5e658d9980ea906725fb6c.

Reply:
Acknowledged.